kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/exception/job/MissRequireTargetDatabaseException.java

```diff
@@ -20,6 +20,9 @@
 import org.apache.shardingsphere.data.pipeline.core.exception.PipelineSQLException;
 import org.apache.shardingsphere.infra.util.exception.external.sql.sqlstate.XOpenSQLState;
 
+/**
+ * Miss requre target database exception.
+ */
 public final class MissRequireTargetDatabaseException extends PipelineSQLException {
```

Comment:
`MissRequireTargetDatabaseException` could be `MissingRequiredTargetDatabaseException`